learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md

```diff
@@ -62,6 +62,8 @@ The VS Code Ballerina extension gives you the same debugging experience as the c
 
 Then, you can start a program, test, or remote debug session as shown below.
 
+>**Info:** The working directory will always be the package root if the user starts the debug session through VS Code. Users can use remote debugging for alternative working directories.
```

Comment:
```suggestion
>**Info:** If you launch the debug session through VS Code, the working directory will always be the package root. However, you can use remote debugging for alternative working directories.
```

Reply:
>

Reply:
@praneesha How about `>**Info:** If you launch the debug session through VS Code, the working directory will be the Ballerina package root. However, you can use remote debugging for alternative working directories.`

Reply:
+1

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-dev-website/pull/2151/commits/9638a4849dbb2b96e940cfa945c9e12d58c00b23